static/css/legacy-datatables.less

```diff
@@ -30,7 +30,6 @@
   position: relative;
   z-index: @z-index-level-1;
   margin: 15px 0;
-  float: left;
```

Comment:
Are you sure this doesn't break any other dataTables (I don't know the answer)?
If you're not confident, I'd suggest keeping this and adding a more specific CSS selector inside static/css/components/editions.less to override this rule.

Reply:
@jdlrobson this change is no longer necessary in the latest changes I made, so I'll leave it unchanged in the next commit 👍